pkg/worker/container_logger.go

```diff
@@ -0,0 +1,117 @@
+package worker
```

Comment:
If we only capture those few logs in the image client surrounding the blobfs caching, do we need to refactor this out to a separate file / logger? Feels like overkill

Reply:
I wasn't sure if we would want to add similar logging in other places in the future. In any case, I've removed this refactoring for now.